<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dialog</class>
 <widget class="QDialog" name="dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>jedit 정보</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>351</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>jedit - jacker editor</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>351</width>
     <height>171</height>
    </rect>
   </property>
   <property name="text">
    <string>jedit - jacker editor
버전 : 0.17.01.Windows
뭉탱이로 있다가 유리게슝 아니 그냥

201758038 전재호
acmicpc.net/user/jaehoo1
Server Management jacker
jaehoo8936@naver.com

made by PyQt</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_confirm">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>260</y>
     <width>80</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>화긴</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
